broker-cli/commands/orderbook.js

```diff
@@ -107,11 +107,18 @@ function createUI (market, asks, bids) {
  * @param {Logger} logger
  */
 async function orderbook (args, opts, logger) {
-  const { market, rpcAddress } = opts
+  const { market, rpcAddress, json } = opts
   const request = { market }
 
   try {
     const brokerDaemonClient = new BrokerDaemonClient(rpcAddress)
+
+    if (json) {
```

Comment:
Per our conversation, I think there should be an option for a static orderbook that still displays the UI (#387), and I would consider the json option only valid with that option as well. But for this PR I think it's ok to merge.